eng/Packages.Data.props

```diff
@@ -175,7 +175,7 @@
     <PackageReference Update="Microsoft.Azure.ApplicationInsights.Query" Version="1.0.0" />
     <PackageReference Update="Microsoft.AspNetCore" Version="2.2.0" />
     <PackageReference Update="Microsoft.AspNetCore.Mvc.Testing" Version="2.2.0" />
-    <PackageReference Update="Microsoft.AspNetCore.Server.Kestrel" Version="2.2.0" />
+    <PackageReference Update="Microsoft.AspNetCore.Server.Kestrel" Version="2.1.3" />
```

Comment:
Did you intend to downgrade this?

Reply:
Yes we didn't need 2.2 since I moved this to a more central dependency 2.2 would have required upgrading a bunch of other dependencies.

Reply:
I defer to you and @pakrym  on this one.